### PR TITLE
Re-adds freeform/purge boards to all AI uploads, removes from spawners

### PR DESCRIPTION
**Pull request by Son-of-Space**
## About The Pull Request

Re-adds the freeform boards to an open table in all the AI uploads, re-adds the purge boards to the harmful table in all uploads, and removes them from their associated spawners so that all stations have them as roundstart guaranteed spawns.

## Why It's Good For The Game
Gives players the agency to shoot themselves in the foot by adding a poorly worded law to the AI or unleashing it, as god intended.

Joking aside, enabling players to add on to the AIs laws creatively has always been a staple of the AI's laws mechanic and removing it for the sake of randomization was a poor idea. This change re-opens the door to chaos and/or fun storytelling.

## Changelog

:cl:
balance: The AIs freeform and purge law boards have been returned as a static staple on all maps.
/:cl:

### CODE REVIEW

**Comment by timothymtorres** [comment]
Before I added the random spawners for AI uploads, all the maps had different lawsets.  Not every map had freeform or freeformcore.  Every map had a set of asimov + reset boards.  The only difference between a purge board and a reset board is that the purge deletes all the core lawsets.  You can also achieve this affect by uploading any core module lawset. (all prior core lawsets get deleted and replaced with the new ones)

My randomized odds for the spawners gave each round the following chances to spawn:
- freeform (~20%)
- purge (~25%)
- freeformcore (~50%)

I do not support this PR and do not think any of these should be guaranteed spawns.  That's also why I went through the trouble of adding every board to the RnD circuit printer as an alternative way of obtaining them all. (which this PR detracts from)  If you are still going to go through with this I would recommend changing the weighted chances on the spawners so that freeform/purge is higher.

**Comment by zxaber** [comment]
> That's also why I went through the trouble of adding every board to the RnD circuit printer as an alternative way of obtaining them all.

This requires mid-ish game tech (including the dissection experiment), as wll as diamond *and* bluespace mats.

Outside of the alt-laws station trait, it should absolutely be within the captain's right to make a meme lawset that may or may not backfire horribly.

**Comment by tralezab** [comment]
> I do not support this PR and do not think any of these should be guaranteed spawns. That's also why I went through the trouble of adding every board to the RnD circuit printer as an alternative way of obtaining them all. (which this PR detracts from) If you are still going to go through with this I would recommend changing the weighted chances on the spawners so that freeform/purge is higher.
a
Freeform,

File: code/game/objects/effects/spawners/random/ai_module.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	spawn_loot_count = 3
 	spawn_loot_split = TRUE
 
-// AI uploads have both ai_module/reset and ai_module/core/full/asimov directly mapped in
+/// AI uploads have the ai_module/reset , ai-module/supplied/freeform , ai_module/reset/purge , and ai_module/core/full/asimov directly mapped in
```
Comment (san7890) [inline code/game/objects/effects/spawners/random/ai_module.dm:9]:
```suggestion
/// AI uploads have the ai_module/reset , ai_module/supplied/freeform , ai_module/reset/purge , and ai_module/core/full/asimov directly mapped in
```

pedantics

**Pull request merged**